### DISCUSSION TITLE
Do we have possibility to send URL / File Content directly instead of sending File

### DISCUSSION THREAD

**Discussion by sourabh-tiwari-swi** [dodona-edu/dolos#950] [Q&A] [answered]
Do we have possibility to send URL / File Content directly instead of sending File

**Comment by rien** [comment, marked as answer]
You can currently analyse files in three formats:
1. listing the files directly
2. using an info.csv file with the list of files on each row
3. using a ZIP-file with such an info.csv file in it

The only way to start an analysis is currently through the command line using the CLI client.

However, we're working on a Web server and API so you can upload and analyse your files without installing anything. We're planning on releasing this soon, so stay tuned.

Does this answer your question?
